packages/@ember/-internals/glimmer/lib/components/link-to.ts

```diff
@@ -657,22 +638,6 @@ if (EMBER_MODERNIZED_BUILT_IN_COMPONENTS) {
       configurable: true,
       enumerable: false,
       get: function isDisabled(this: LinkTo): boolean {
-        if ('disabledWhen' in this.args.named) {
-          deprecate(
-            'Passing the `@disabledWhen` argument to <LinkTo> is deprecated. ' +
-              'Use the `@disabled` argument instead.',
-            false,
-            {
-              id: 'ember.link-to.disabled-when',
-              for: 'ember-source',
-              since: {},
-              until: '4.0.0',
-            }
-          );
-
-          return Boolean(this.named('disabledWhen'));
-        }
-
         return superGetter.call(this);
       },
     });
```

Comment:
Same here?

Reply:
Not familiar with this code: would `isDisabled` still work if I remove it?

Reply:
I think so. And even a few lines above this can get dropped I believe.

You could poke around for a test of `isDisabled` to confirm the removal doesn't regress anything.